coordinator/src/test/java/com/tencent/rss/coordinator/ClientConfManagerTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.coordinator;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.Path;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.tencent.rss.storage.HdfsTestBase;
+
+import static java.lang.Thread.sleep;
+import static org.junit.Assert.assertEquals;
+
+public class ClientConfManagerTest extends HdfsTestBase {
+
+  @Before
+  public void setUp() {
+    CoordinatorMetrics.register();
+  }
+
+  @Test
+  public void testLocal() throws Exception {
```

Comment:
the case should include:
1. update conf
2. delete conf
3. remove conf file

Reply:
ok